### DISCUSSION TITLE
Remove #help, #videogames, #mapping, #modreqs

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#26049] [Ideas]
... INSTEAD let it redirect to forums to get more people in there. Named channels aren't really fulfilling their purpose anymore, and if they do, it's most likely not active at all. Current small communities in these channels do be chilling in other channels aswell, it will only hurt them for a short amount of time

#help - Help forums (https://osu.ppy.sh/community/forums/topics/create?forum_id=5)
#videogames - Video Games forum (https://osu.ppy.sh/community/forums/topics/371494?n=1)
#mapping - Mapping Discussion forum (https://osu.ppy.sh/community/forums/topics/1274956?n=1)
#modreqs - Modding Queues forum (https://osu.ppy.sh/community/forums/topics/726725?n=1)

**Comment by bdach** [comment]
not really the correct place for this, but sure
